tutorial/westeros/westeros_firm_capacity.ipynb

```diff
@@ -196,6 +248,36 @@
     "p.plot_activity(baseyear=True, subset=['coal_ppl', 'wind_ppl'])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The difference between the two scenarios can be seen when comparing the capacity.  Whereas in the 'emission_bound' scenario, coal capacity is quickly phased out, the 'firm_capacity' scenario shows, that there is a substantial amount of capacity required from the coal_ppl in order for the wind_ppl to achieve such high market share levels."
```

Comment:
Suggest to restructure as for *Activity* - it's odd when the explanation text and the section are not aligned.